src/Components/ResultTable/ResultTable.js

```diff
@@ -2,9 +2,11 @@ import React, { useMemo } from 'react'
 import { useTable, useSortBy } from 'react-table'
 
 import './ResultTable.css'
-import { getCSV } from './TableUtils'
+import { getInvisibleStyle } from './TableUtils'
 
-function ResultTable({ energies, values, stoppingPowerUnit }) {
+// Once #
```

Comment:
What does it mean ?

Reply:
Comment to deleted function. Removed it.